engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -21,10 +27,21 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
                               restarting: Boolean,
                               serviceRegistryActor: ActorRef,
                               jobStoreActor: ActorRef,
+                              backendName: String,
                               callCachingMode: CallCachingMode) extends LoggingFSM[EngineJobExecutionActorState, EJEAData] with WorkflowLogging {
 
   override val workflowId = executionData.workflowDescriptor.id
 
+  val tag = s"EJEA [UUID(${workflowId.shortString}/${jobKey.call.fullyQualifiedName}:${jobKey.index.fromIndex}(attempt ${jobKey.attempt}))]"
+
+  // There's no need to check for a cache hit again if:
+  // - We got preempted
```

Comment:
I was hoping I could also add "restarted calls" - but unfortunately the WF only tells us whether we're in a restarted workflow, which isn't good enough.

OTOH, the impact isn't actually too bad since most restarted jobs will be short-circuited already by the JobStore